storage/innobase/log/log0recv.cc

```diff
@@ -238,29 +238,6 @@ static bool recv_writer_is_active() {
 
 #ifndef UNIV_HOTBACKUP
 
-<<<<<<< HEAD
-||||||| 824e2b40640
-/** Reads a specified log segment to a buffer.
-@param[in,out]  log             redo log
-@param[in,out]  buf             buffer where to read
-@param[in]      start_lsn       read area start
-@param[in]      end_lsn         read area end
-@return lsn up to which data was available on disk (ideally end_lsn) */
-static lsn_t recv_read_log_seg(log_t &log, byte *buf, lsn_t start_lsn,
-                               lsn_t end_lsn);
-
-=======
-/** Reads a specified log segment to a buffer.
-@param[in,out]  log             redo log
-@param[in,out]  buf             buffer where to read
-@param[in]      start_lsn       read area start
-@param[in]      end_lsn         read area end
-@return lsn up to which data was available on disk (ideally end_lsn)
-or zero in case of error */
-static lsn_t recv_read_log_seg(log_t &log, byte *buf, lsn_t start_lsn,
-                               lsn_t end_lsn);
-
->>>>>>> mysql-8.4.0
```

Comment:
recv_read_log_seg is competely removed by upstream? or removed by us?

Reply:
this is a misleading diff. recv_read_log_seg() is present in both. 
It is just that we make it non-static. the reason PXB redo log parser calls this function to parse redo log.

Closing this conversation as resolved